workflows/prognostic_c48_run/tests/_regtest_outputs/test_regression.test_fv3run_diagnostic_outputs_check_variables[keras].out

```diff
@@ -0,0 +1,37 @@
+air_temperature: 41ccaafb84a9f1b3e2195eb703974c5c
```

Comment:
How does the list of variables specified in the test script to be written to `diags.zarr` get expanded to this larger list?

Reply:
I modified "diags.zarr" to save all the available diagnostics, and then looped through them in the test. If we somehow delete one by accident it will be missing in the regression report and the test will fail.

Reply:
OK thanks. See my comment about the "other_variables" list in the test script then.